### PR TITLE
[Docs] Landing page update: Added Galaxy Explorer. Example scene screenshots update

### PR DESCRIPTION
**Pull request by cre8ivepark**
## Overview
Landing page update: Added Galaxy Explorer. Added contribution flow diagram to help creators who are not familiar with the GitHub.

2019-11-19 17_53_49-README md - MRTK-Public-Microsoft - Visual Studio Code

2019-11-19 17_52_43-CONTRIBUTING md - MRTK-Public-Microsoft - Visual Studio Code

## Preview page
https://github.com/cre8ivepark/MixedRealityToolkit-Unity/tree/prerelease/2.2.0_stabilization_DocsLandingpage1119

https://github.com/cre8ivepark/MixedRealityToolkit-Unity/blob/prerelease/2.2.0_stabilization_DocsLandingpage1119/Documentation/Contributing/CONTRIBUTING.md

### CODE REVIEW

**Comment by cre8ivepark** [comment]
Any feedback on the contribution flow diagram would be appreciated!

**Comment by julenka** [comment]
It would be great if we could add in between c and d the following: Read through the [MRTK coding guidelines](https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/Contributing/CodingGuidelines.html). 

Then for step 3, say "when writing pull request description, use [MRTK pull request guidelines](https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/Contributing/PullRequests.html) to help guide you"

File: README.md [hunk 1]
```diff
@@ -79,11 +79,11 @@ See [Examples Hub README page](Documentation/README_ExampleHub.md) to learn abou
 [![Example Scene](Documentation/Images/MRTK_ExamplesHub.png)](Documentation/README_HandInteractionExamples.md)
 
 
-# Sample App
+# Sample apps made with MRTK
+| [![Periodic Table of the Elements](Documentation/Images/MRDL_PeriodicTable.jpg)](https://medium.com/@dongyoonpark/bringing-the-periodic-table-of-the-elements-app-to-hololens-2-with-mrtk-v2-a6e3d8362158)| [![Periodic Table of the Elements](Documentation/Images/MRTK_GalaxyExplorer.jpg)](https://docs.microsoft.com/en-us/windows/mixed-reality/galaxy-explorer-update)|
```
Comment (wiwei) [inline README.md]:
```suggestion
| [![Periodic Table of the Elements](Documentation/Images/MRDL_PeriodicTable.jpg)](https://medium.com/@dongyoonpark/bringing-the-periodic-table-of-the-elements-app-to-hololens-2-with-mrtk-v2-a6e3d8362158)| [![Galaxy Explorer](Documentation/Images/MRTK_GalaxyExplorer.jpg)](https://docs.microsoft.com/en-us/windows/mixed-reality/galaxy-explorer-update)|
```

Comment (wiwei) [inline README.md]: Super nit: The two images are SLIGHTLY different sizes from each other. My OCD is somewhat strong but here's what I mean:

image

**Comment by wiwei** [comment]
For the contribution flow image, I'm a little conflicted about having it be an image, vs just having it be text. A few thoughts here:

1. Images are really awesome for cases where text isn't sufficient in explaining, though in this case, this is primarily text content - this is probably something where we could have a text flow (i.e. 1, 2, 3, i, ii, iii, iv, 4...)
1. Images are harder to update than text - i.e. if we change our branch naming, or if we want to change the flow at certain times (i.e. stabilization), this is harder than just updating specific words/text.

**Comment by wiwei** [comment]
Another note - I think for the name of the branch (i.e. mrtk_development_featurename) that's not actualllllly necessary (i.e. people can name it whatever they want). It would probably be more accurate to say like create a working branch off of mrtk_development (e.g. "git checkout -b yourawesomefeaturewithabettername)

The wording there is currently kinda imperative

**Comment by cre8ivepark** [comment]
@wiwei I'll try with a multi-column table

**Comment by Troy-Ferrell** [comment]
> For the contribution flow image, I'm a little conflicted about having it be an image, vs just having it be text. A few thoughts here:
> 
> 1. Images are really awesome for cases where text isn't sufficient in explaining, though in this case, this is primarily text content - this is probably something where we could have a text flow (i.e. 1, 2, 3, i, ii, iii, iv, 4...)
> 2. Images are harder to update than text - i.e. if we change our branch naming, or if we want to change the flow at certain times (i.e. stabilization), this is harder than just updating specific words/text.

I love the look of the flow image...but yea hard to update, also more importantly can't add hyperlinks. If it was designed in HTML/CSS than it would be perfect for all worlds....but I'm sure that is a lot more work

File: README.md (hunk 1 above)
Comment (cre8ivepark) [reply README.md]: @wiwei Acutally, they have the same size but this is a limitation of the GitHub's markdown table :) There is no way to control the width of the table and image width. It is controlled by the amount of the text in each column.

**Comment by cre8ivepark** [comment]
Removed the flow diagram for this PR - need to find a better way. I found that we have a link to the GitHub guideline page already.

**Comment by cre8ivepark** [comment]
Updated three screenshot image for the eye tracking example scenes.

**Pull request merged**